.flake8

```diff
@@ -0,0 +1,7 @@
+[flake8]
+exclude=.git,__pycache__,docs/source/conf.py,old,build,dist,venv,__init__.py,./lib/*,./bin/*
```

Comment:
We have pretty much all of our code in the bin directory, won't this stop it from being checked? (I'm not sure why we have everything inside bin, should probably be taken out or renamed to something that makes more sense.)